src/main/java/frc/robot/commands/Presets/AmpPreset.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands.Presets;
+
+
+import edu.wpi.first.wpilibj2.command.Command;
+import static frc.robot.RobotContainer.*;
+
+public class AmpPreset extends Command{
+    public AmpPreset(){
+        addRequirements(PIVOT);
+    }
+ 
+    
+    @Override
+    public void initialize() {
+       PIVOT.AmpPreset();
```

Comment:
Are any plans to do more here? this could a pivot subsystem command. same for subwoofer